awx/ui_next/src/screens/Job/Jobs.jsx

```diff
@@ -42,6 +41,12 @@ function Jobs({ i18n }) {
     [i18n]
   );
 
+  function TypeRedirect({ view }) {
+    const { id } = useParams();
+    const { path } = useRouteMatch();
```

Comment:
On line 22, you could replace `const match = useRouteMatch();` with this line, and replace every instance of `${match.path}` with `path`.